Python/jit.c

```diff
@@ -287,6 +288,7 @@ patch(unsigned char *base, const Stencil *stencil, uintptr_t patches[])
                 if (i + 1 < stencil->holes_size &&
                     (next_hole->kind == HoleKind_ARM64_RELOC_GOT_LOAD_PAGEOFF12 ||
                      next_hole->kind == HoleKind_IMAGE_REL_ARM64_PAGEOFFSET_12L ||
+                     next_hole->kind == HoleKind_ARM64_RELOC_BRANCH26 ||
```

Comment:
And then this should be removed. The AArch64 relocations sometimes come in pairs: one relocation for 21 high bits of the address, and another for the 12 low bits of the address. When they occur together, they can sometimes be optimized (which is what we're checking here).

Notice how all of the cases for `hole` here have "21" in their name, and all of the `next_hole` values we're checking for have "12" in their name.